tests/test_matrix_utils_connect_rooms.py

```
@@ -0,0 +1,295 @@
+    mock_client.should_upload_keys = False
+    mock_client.get_displayname = AsyncMock(
+        return_value=SimpleNamespace(displayname=None)
+    )
+    mock_client.close = AsyncMock()
+
+    monkeypatch.setattr(
+        "mmrelay.matrix_utils.AsyncClient",
+        lambda *_args, **_kwargs: mock_client,
+    )
+    monkeypatch.setattr(
+        "mmrelay.matrix_utils._create_ssl_context", lambda: MagicMock(), raising=False
```

Ruff (0.15.9)

[warning] Lambda may be unnecessary; consider inlining inner function

Inline function call

(PLW0108)